### ISSUE TITLE
BYOL-1.0 CIFAR-10 top-1 acc not reproducible with pre-trained weight

### ISSUE THREAD

**Issue by daisukelab** [Spijkervet/BYOL#2]
Hello @Spijkervet, thank you for sharing BYOL implementation with CIFAR-10 result which is supposed to be reproducible.

My problem is, the result top-1 accuracy 0.832 doesn't reproduce in my local environment even with using your `1.0/resnet18-CIFAR10-final.pt`. As far as I could train, the result shows the same. My result is around 0.43 for both your pre-trained model and my locally trained one.

I'm using torch==1.5.0. I appreciate if you could take a quick look what's wrong.

Log:
```
$ ~/lab/BYOL/BYOL-1.0$ python logistic_regression.py --model_path ./resnet18-CIFAR10-final.pt
--
Downloading https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz to ./datasets/cifar-10-python.tar.gz
99%\|███████████████████████████▋ \| 168206336/170498071 [00:11<00:00, 16383016.35it/s]
Extracting ./datasets/cifar-10-python.tar.gz to ./datasets
Files already downloaded and verified
### Creating features from pre-trained model ###
Step [0/65] Computing features...
Step [5/65] Computing features...
Step [10/65] Computing features...
Step [15/65] Computing features...
Step [20/65] Computing features...
Step [25/65] Computing features...
170500096it [00:30, 16383016.35it/s]
Step [30/65] Computing features...
Step [35/65] Computing features...
Step [40/65] Computing features...
Step [45/65] Computing features...
Step [50/65] Computing features...
Step [55/65] Computing features...
Step [60/65] Computing features...
Features shape (49920, 512)
Step [0/13] Computing features...
Step [5/13] Computing features...
Step [10/13] Computing features...
Features shape (9984, 512)
Epoch [0/300]: Loss/train: 2.257830924987793 Accuracy/train: 0.16569661458333332
Epoch [1/300]: Loss/train: 2.083581094741821 Accuracy/train: 0.24340494791666667
Epoch [2/300]: Loss/train: 2.0132872915267943 Accuracy/train: 0.2754361979166667
Epoch [3/300]: Loss/train: 1.9688853120803833 Accuracy/train: 0.2946354166666667
Epoch [4/300]: Loss/train: 1.9352103281021118 Accuracy/train: 0.31092447916666666
Epoch [5/300]: Loss/train: 1.9084804725646973 Accuracy/train: 0.32227213541666666
Epoch [6/300]: Loss/train: 1.8860071468353272 Accuracy/train: 0.33306640625
  :
Epoch [295/300]: Loss/train: 1.5378710174560546 Accuracy/train: 0.4568359375
Epoch [296/300]: Loss/train: 1.537794508934021 Accuracy/train: 0.4569140625
Epoch [297/300]: Loss/train: 1.5377185583114623 Accuracy/train: 0.45677734375
Epoch [298/300]: Loss/train: 1.5376431322097779 Accuracy/train: 0.456796875
Epoch [299/300]: Loss/train: 1.537568416595459 Accuracy/train: 0.45681640625
### Calculating final testing performance ###
Final test performance: Accuracy/test: 0.423828125
```

**Comment by daisukelab** [comment]
Quickly debugged and found that model wasn't loaded actually. 😅
https://github.com/Spijkervet/BYOL/blob/master/logistic_regression.py#L89

Uncomment this line reproduced your result. Thank you!

I'll leave this open in case you might not be aware of potentially unintentional commenting out...

P.S. my locally trained model also marked 0.8312%, great!

**Comment by Spijkervet** [comment]
Thanks! I fixed it in the last commit 👍
